nvidia/advanced/driver_packages/customize_filesystem.rst

```diff
@@ -0,0 +1,108 @@
+Customize System FileSystem
+###########################
+
+Customize file system on **Host PC**. 
+
+1. Prepare Filesystem
+^^^^^^^^^^^^^^^^^^^^^
+
+| Driver package allows users to customize their filesystem. 
+| Driver package provides a script (create_image.sh) to create a board specific image base on several rootfs selection, here's what we support.
+
+So you need to prepare these items:
+
+    * Driver package
+    * Rootfs Binaries
+
+Download Rootfs Binaries
+------------------------
+
+**nvidia_sample_rootfs - Nvidia provided sample file system.**
```

Comment:
This line can be modified as the following:
The file system we use here is NVIDIA Sample Filesystem.

Reply:
OK